### PR TITLE
Bugfixes/sim query and frontend fixes

### PR DESCRIPTION
**Pull request by JaviEst**

### CODE REVIEW

File: lexos/frontend/html/helpers/analyze-template.html [hunk 1]
```diff
@@ -70,23 +70,23 @@
             </div>
 
             <div class="section-body">
-                <label for="most-frequent-words-checkbox" class="hidden">Enable Most Frequent Words</label>
-                <input id="most-frequent-words-checkbox" name="enable_most_frequent_words"
-                        type="checkbox">
+                <!--<label for="most-frequent-words-checkbox" class="hidden">Enable Most Frequent Words</label>-->
```
Comment (Myles-Trevino) [inline lexos/frontend/html/helpers/analyze-template.html:73]: Remove commented code.

File: lexos/frontend/html/helpers/analyze-template.html [hunk 2]
```diff
@@ -70,23 +70,23 @@
...
+                <label for="most-frequent-words-input">Use the top
                 <input id="most-frequent-words-input"
                         name="most_frequent_words" value="100" type="text" spellcheck="false"
-                        autocomplete="off"><h3>terms</h3>
+                        autocomplete="off">terms</label>
                 <span id="most-frequent-words-tooltip-button" class="tooltip-button">?</span><br>
-                <label for="minimum-occurrences-checkbox" class="hidden">Enable Minimum Occurrences</label>
-                <input id="minimum-occurrences-checkbox" name="enable_minimum_occurrences"
-                        type="checkbox">
+                <!--<label for="minimum-occurrences-checkbox" class="hidden">Enable Minimum Occurrences</label> -->
```
Comment (Myles-Trevino) [inline lexos/frontend/html/helpers/analyze-template.html:82]: Remove commented code.

File: lexos/frontend/html/statistics.html [hunk 3]
```diff
@@ -68,22 +68,20 @@
 
                 <div class="section-body">
 
-                    <label for="most-frequent-words-checkbox" class="hidden">Enable Most Frequent Words</label>
-                    <input id="most-frequent-words-checkbox" name="enable_most_frequent_words" type="checkbox">
-                    <span></span>
-                    <label for="most-frequent-words-input">Use the top</label>
+                    <!--<label for="most-frequent-words-checkbox" class="hidden">Enable Most Frequent Words</label>-->
```
Comment (Myles-Trevino) [inline lexos/frontend/html/statistics.html:71]: Remove commented code.

File: lexos/frontend/html/statistics.html [hunk 4]
```diff
@@ -68,22 +68,20 @@
...
-                        spellcheck="false" autocomplete="off">
-                    terms
+                        spellcheck="false" autocomplete="off">terms</label>
                     <span id="most-frequent-words-tooltip-button" class="tooltip-button">?</span><br>
 
-                    <label for="minimum-occurrences-checkbox">Minimum Occurrences</label>
-                    <input id="minimum-occurrences-checkbox" name="enable_minimum_occurrences" type="checkbox">
-                    <span></span>
-                    <label for="minimum-occurrences-input">Must be in</label>
+                    <!--<label for="minimum-occurrences-checkbox">Minimum Occurrences</label>-->
```
Comment (Myles-Trevino) [inline lexos/frontend/html/statistics.html:79]: Remove commented code.

File: lexos/frontend/html/rolling-window.html [hunk 5]
```diff
@@ -83,7 +82,7 @@
 
                 <label for="window-words"><input id="window-words" name="window_type" value="Words" type="radio" checked><span></span>Words</label><br>
                 <label for="characters"><input id="characters" name="window_type" value="Characters" type="radio"><span></span>Characters</label><br>
-                <label for="lines"><input id="lines" name="window_type" value="Lines" type="radio"><span></span>Lines</label>
+                <label><input id="lines" name="window_type" value="Lines" type="radio"><span></span>Lines</label>
```
Comment (Myles-Trevino) [inline lexos/frontend/html/rolling-window.html:85]: Add the `for="lines"` attribute back to the label.

**Review by Myles-Trevino — CHANGES_REQUESTED** [review]
Consider removing the code that is commented out and make sure you want to remove the `for="lines"` attribute from the label in Rolling Window.

File: lexos/frontend/html/rolling-window.html (hunk 5 above)
Comment (WeaselPeasel) [reply lexos/frontend/html/rolling-window.html:85]: There is an issue where we cannot select the lines radio button if the `for="lines"` attribute is included (fails on both Chrome and Firefox). Likely some issue buried in our CSS. Is this something we should absolutely add for this commit?

Comment (WeaselPeasel) [reply lexos/frontend/html/rolling-window.html:85]: There was another section that used `id="lines"`, so once I changed the id name and the `label for=""` attributes, we got it working.

**Pull request merged**